docs/api/lib-scheduler.adoc

```diff
@@ -0,0 +1,471 @@
+[frame="none"]
+[grid="none"]
+|===
+| Name   | Type   | Description
+| params | object | JSON with the parameters
+
+[%header,cols="1%,1%,1%,97%a", options="header"]
+[frame="topbot"]
+[grid="none"]
+[caption=""]
+.Properties
+!===
+! Name ! Type ! Attributes ! Description
+
+! name ! string ! required ! name of the job.
+
+
+!===
+|===
+[.lead]
+Returns
+
+*object* : (<<#scheduled_job,`ScheduledJob`>>) Detail information for the job.
+Or null if the job could not be found.
+
+[.lead]
+Example
+
+.Obtains details of a scheduled job:
+```js
+var job = schedulerLib.get('myjob');
+
+if (job) {
+  log.info('Job is found: ' + JSON.stringify(job));
+} else {
+  log.info('Job not found');
+}
+```
+
+.Return value:
+```js
+var expected = {
+    'name': 'myjob',
+    'descriptor': 'com.enonic.app.myapp:mytask',
+    'description': 'cron job description',
+    'enabled': true,
+    'config': {
+        'a': 1,
+        'b': 'value',
+        'c': {
+            'd': {
+                'e': 3.6,
+                'f': true
+            }
+        }
+    },
+    'user': 'user:system:user',
+    'creator': 'user:system:creator',
+    'modifier': 'user:system:creator',
+    'createdTime': '2016-11-02T10:36:00Z',
+    'modifiedTime': '2016-11-02T10:36:00Z',
+    'lastRun': '2021-02-25T10:44:33.170079900Z',
+    'lastTaskId': 'task-id',
+    'schedule': {
+        'value': '* * * * *',
+        'type': 'CRON',
+        'timezone': 'GMT+2:00'
+    }
+};
+```
+
+=== list
+
+Returns the list of scheduled jobs.
+
+[.lead]
+Returns
+
+*Array* : (<<#scheduled_job,`ScheduledJob[]`>>) List of all scheduled jobs.
+
+[.lead]
+Example
+
+.Obtains list of existing jobs:
+```js
+var jobs = schedulerLib.list();
+```
+
+.Return value:
+```js
+var expected = [
+  {
+        'name': 'job1',
+        'descriptor': 'appKey:task',
+        'description': 'job description',
+        'enabled': true,
+        'config': {
+            'a': 1
+        },
+        'user': 'user:system:user',
+        'creator': 'user:system:creator',
+        'modifier': 'user:system:creator',
+        'createdTime': '2016-11-02T10:36:00Z',
+        'modifiedTime': '2016-11-02T10:36:00Z',
+        'lastRun': '2021-02-25T10:44:00.170079900Z',
+        'lastTaskId': 'task-id',
+        'schedule': {
+            'value': '* * * * *',
+            'timeZone': 'GMT+05:30',
+            'type': 'CRON'
+        }
+    },
+    {
+        'name': 'job2',
+        'descriptor': 'appKey:task',
+        'description': 'job description',
+        'enabled': false,
+        'config': { },
+        'user': 'user:system:user',
+        'creator': 'user:system:creator',
+        'modifier': 'user:system:creator',
+        'createdTime': '2021-02-02T10:36:00Z',
+        'modifiedTime': '2021-02-02T10:36:00Z',
+        'schedule': {
+            'value': '2012-01-01T00:00:00Z',
+            'type': 'ONE_TIME'
+        }
+    }
+];
+```
+
+=== create
+
+Creates a job to be scheduled and run in the background.
```

Comment:
Just "_Creates a scheduled job_", otherwise it looks like if the job will run immediately. Or "_Create and schedule a job_"